test/e2e-shared-tests/mcd_config_drift.go

```diff
@@ -294,6 +294,27 @@ func (c configDriftTest) Run(t *testing.T) {
 				assertNodeAndMCPIsRecovered(t, c.ClientSet, c.node, c.mcp)
 			},
 		},
+		{
```

Comment:
**thought:** I'm not sure that this is the correct place for this test to live because it isn't directly related to the Config Drift Monitor. That said, I'm not sure where else it should live.

(To be clear, I'm not asking you to change it as part of this review.)